Description: Fix spelling error in binary. 
Author: Scarlett Clark <sgclark@kde.org> 
Bug: https://bugs.kde.org/show_bug.cgi?id=396676
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Last-Update: 2018-07-19 
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/app/main.cpp
+++ b/app/main.cpp
@@ -168,7 +168,7 @@ int main(int argc, char **argv)
         layoutNameOnStartup = parser.value(QStringLiteral("layout"));
 
         if (!Latte::Importer::layoutExists(layoutNameOnStartup)) {
-            qInfo() << i18nc("layout missing", "This layout doesnt exist in the system.");
+            qInfo() << i18nc("layout missing", "This layout doesn't exist in the system.");
             qGuiApp->exit();
             return 0;
         }
